.github/workflow_templates/create-market-place-pr/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Create PR in operator market place repository
+description: Create PR in redhat-openshift-ecosystem/community-operators-prod from fork
+inputs:
+  token:
+    description: Token that has write access to fork repo
+    required: true
+  version:
+    description: The operator version to be released
+    required: true
+  fork:
+    description: The username of the fork to open PR from
```

Comment:
It should be a user username (in this case I am using mine, but it should be a bot account). So the user (not a ORG) must already have the fork created (created by itself and with the same name as the original repo) to run it

Did you understand now? How should be a better description written to avoid misleading in the future?

Reply:
TODO for myself: add how to create token some where